satpy/composites/__init__.py

```diff
@@ -1236,3 +1236,81 @@ def __call__(self, datasets, optional_datasets=None, **attrs):
 
         return super(SelfSharpenedRGB, self).__call__(
             (red, green, blue), optional_datasets=(high_res,), **attrs)
+##
+class Airmasstr(GenericCompositor):
```

Comment:
Would `AirmassTr` make more sense for this class?

Reply:
this rgb is adapted from EUMETSAT webpage wich is named Tropical Airmass RGB
maybe TropicalAirmass is more sense for this class

Reply:
> TropicalAirmass

Perfect!